python/ray/data/context.py

```diff
@@ -8,6 +8,9 @@
 _default_context: "Optional[DatasetContext]" = None
 _context_lock = threading.Lock()
 
+# The max target block size in bytes for reads and transformations.
+DEFAULT_TARGET_MAX_BLOCK_SIZE = 500 * 1024 * 1024
```

Comment:
Nit: 512 MiB keeps it a power of 2, and is the same as 0.5 GiB
```suggestion
DEFAULT_TARGET_MAX_BLOCK_SIZE = 512 * 1024 * 1024
```

Reply:
Keeping at 500MiB to be a little more human friendly (doesn't matter too much either way).